src/components/NavBar.js

```diff
@@ -0,0 +1,47 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+const NavBar = (props) => {
+  const API =
+    "https://api.themoviedb.org/3/search/movie?api_key=4f298a53e552283bee957836a529baec&language=en-US&page=1&include_adult=false&query=";
+  const [search, setSearch] = useState("");
+  
+  const [movies, setMovies] = useState([]);  
+  const searchHandler = (event) => {
+    event.preventDefault();
+    if (search.trim() !== "") {
+      axios.get(API + search).then((res) => {
```

Comment:
Please don't use simple string concatenation to add the query and instead use Axios.get options.
Please refer to the official axios docs

```suggestion
      axios.get(API, {params: search}).then((res) => {
```